### ISSUE TITLE
TensorRT - TRTEngineOp error

### ISSUE THREAD

**Issue by rhavard** [tensorflow/tensorflow#20120]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: 1.9.0
- **Python version**: 3.5
- **Bazel version (if compiling from source)**: 0.13.0
- **GCC/Compiler version (if compiling from source)**: 5.4.0
- **CUDA/cuDNN version**: CUDA 9.0, cuDNN 7.0
- **GPU model and memory**: GeForce GTX 1070, 8GB
- **Exact command to reproduce**:

### Describe the problem
I've used the code [here](https://github.com/tensorflow/models/tree/master/research/tensorrt) and I am able to run it. My problem is that I can't visualise the graph in Tensorboard anymore (I'm using the `import_pb_to_tensorboard` script) to check the architecture and I can't load it to use it afterwards.

### Source code / logs

```
Traceback (most recent call last):

  File "/tensorflow/tensorflow/python/tools/import_pb_to_tensorboard.py", line 76, in <module>
    app.run(main=main, argv=[sys.argv[0]] + unparsed)

  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/platform/app.py", line 125, in run
    _sys.exit(main(argv))

  File "/tensorflow/tensorflow/python/tools/import_pb_to_tensorboard.py", line 58, in main
    import_to_tensorboard(FLAGS.model_dir, FLAGS.log_dir)

  File "/tensorflow/tensorflow/python/tools/import_pb_to_tensorboard.py", line 49, in import_to_tensorboard
    importer.import_graph_def(graph_def)

  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/util/deprecation.py", line 432, in new_func
    return func(*args, **kwargs)

  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/framework/importer.py", line 418, in import_graph_def
    graph._c_graph, serialized, options)  # pylint: disable=protected-access

tensorflow.python.framework.errors_impl.NotFoundError: Op type not registered 'TRTEngineOp' in binary running on 5b451011e68a. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) 'tf.contrib.resampler' should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.
```

I've seen this error in a post on Stackoverflow ([here](https://stackoverflow.com/a/50461127/9052343)) but that didn't help me. Does anyone have an idea how to solve this issue ?

Thanks.

**Comment by qinyao-he** [comment]
I think you should build tensorflow yourself with TensorRT support.

**Comment by rhavard** [comment]
Thanks for your answer.

I did build Tensorflow from source and with TensorRT support, maybe I didn't specified the correct TensorRT location: I left the default location, I couldn't locate any TensorRT folder after installing it with the debian installation.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @michaelisard: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by qinyao-he** [comment]
@rhavard Try add
`import tensorflow.contrib.tensorrt as trt`
in your code.

**Comment by az-ja** [comment]
@qinyao-he 
I had a similar problem which was solved by your tip. Thank you.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 17 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 33 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by sethtroisi** [comment]
@qinyao-he adding the import also fixed the issue for me.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 59 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by jvishnuvardhan** [comment]
I think it was resolved. Closing due to lack of recent activity. Please open new ticket if you see similar issue. Thanks!

**Comment by Tuanlase02874** [comment]
Solution: add import 

> from tensorflow.python.compiler.tensorrt import trt_convert as trt

link discuss: https://github.com/tensorflow/tensorflow/issues/26525

**Comment by oscarriddle** [comment]
I try to load a TFTRT converted model and execute in C++ version Tensorflow program, and I encountered this problem. How could I analyze TRTEngineOp in C++?

**Comment by kevinjoseph1995** [comment]
@oscarriddle  Any update on the issue you were facing?
